bin/localnet/Dockerfile-geth

```diff
@@ -0,0 +1,14 @@
+FROM frolvlad/alpine-glibc
```

Comment:
I see this as separate repo

Reply:
👍 Moved to: https://github.com/MysteriumNetwork/geth